### ISSUE TITLE
After changing wanted quality, show did not download until doing a manual search

### ISSUE THREAD

**Issue by suprdave23** [SiCKRAGETV/sickrage-issues#993]
Sickrage dev branch pull from 2 weeks ago. Not including logs as this is just a question.

I noticed with the latest Hells Kitchen that came out an HDTV version had not been posted, however an SDTV version had. I changed my wanted quality from HDTV to "Any" in the show's setting. However, during the next Daily Search the episode was still not picked up. I ended up having to manually click the magnifying glass next to the missing episode (which was in Wanted state) in order for it to find the episode and download it.

Is this expected behavior? Or should it have picked up the SDTV during the next Daily Search on its own without me having to perform a manual search?

Thanks!

**Comment by fernandog** [comment]
Should have picked. Please send full logs

**Comment by fernandog** [comment]
Also all informations asked in guidelines

**Comment by suprdave23** [comment]
Last commit I am running: 
C:\SickRage>git show
commit 68cd0fa18e3ae75a368728aeaaae7913caf13db4
Merge: e7349b3 d7106ef
Author: Alexandre Beloin <alexandre.beloin@gmail.com>
Date:   Sun Mar 1 20:28:16 2015 -0500

    Merge pull request #1445 from abeloin/patch_restart_v2

    Redone restart.js



Logs: http://pastebin.com/XHmys3Rh

About 8am is when I changed the quality of Hell's Kitchen to "Any". The torrent it picked up was submitted to KAT about 15 hours ago, so it was there after I had already changed the quality.

I wonder if it's a caching thing again?

**Comment by fernandog** [comment]
@suprdave23 why didn't you updated SR?

**Comment by suprdave23** [comment]
@fernandog I have not been keeping up with the updates because that commit has been working well for me - no more bugs until today.

If there are changes that have changed some of the aspects of SR that could account for this issue I'll be happy to update and let you know if any further problems are seen.

**Comment by fernandog** [comment]
@suprdave23 I suggest you to update. Check the changelog

we can't debug/test SR in older versions.

https://sickrage.tv/forums/forum/main-category/latest-news/19525-enjoy-the-new-update-tons-of-bug-fixes-and-a-lot-of-new-features?p=20315#post20315

**Comment by suprdave23** [comment]
@fernandog OK, I have updated to the latest commit in dev. I recreated the issue by setting the episode back to "Wanted" status and "HD" in order to make it not find a good episode (remember the episode I want is only in SD), then after it was done processing the backlog, I set it back to "Any" quality. It again failed to find the episode during daily search. 

Is it maybe because since I have updated and restarted SR (after the update), the daily search no longer contains older episodes that have been posted?

Log: http://pastebin.com/f7NkWtxP

The "Saving show info to database" at the top is when I changed the show to want Any quality.

**Comment by fernandog** [comment]
@suprdave23 as always: full logs

**Comment by suprdave23** [comment]
@fernandog Here's the entire log for today: https://onedrive.live.com/redir?resid=8674D31BA2E0C41D!123&authkey=!AFgNLD4LfiS494E&ithint=file%2clog

**Comment by fernandog** [comment]
@suprdave23 there is nothing in the log about searching

what did you do first: set to wanted or changed quality?

**Comment by suprdave23** [comment]
@fernandog Honestly I don't remember, so I've done it again. The steps I took are:

* Set episode to "Wanted" status
* Change episode quality from HDTV to "Any" 
* Force a "daily search"
* Episode was not downloaded


If I do it the other direction, change the quality first and *then* set it to "Wanted" status it picks it up immediately due to it processing the backlog. So "Daily Search" does not pick it up, but backlog does.

The log looks exactly the same as what I gave you before, so I haven't included it.
